config/settings/local.py

```diff
@@ -26,6 +26,22 @@
 EMAIL_BACKEND = env(
     "DJANGO_EMAIL_BACKEND", default="django.core.mail.backends.console.EmailBackend"
 )
+ANYMAIL = {
+    "SENDGRID_API_KEY": env("SENDGRID_API_KEY"),
+    "EMAIL_HOST": env("EMAIL_HOST"),
+    "EMAIL_HOST_USER": env("EMAIL_HOST_USER"),
+    "EMAIL_HOST_PASSWORD": env("SENDGRID_API_KEY"),
+    "EMAIL_PORT": env("EMAIL_PORT"),
+    "EMAIL_USE_TLS": env("EMAIL_USE_TLS"),
+    "SENDGRID_API_URL": env("SENDGRID_API_URL", default="https://api.sendgrid.com/v3/"),
+    # "SENDGRID_API_URL": env("SENDGRID_API_URL", default="https://api.sendgrid.com/v3/mail/send"),
```

Comment:
When using the api endpoint you only need the Sengrid api_key and the api_url... so u can take the rest out